### DISCUSSION TITLE
多应用打包后background-image路径错误，img标签路径正常

### DISCUSSION THREAD

**Discussion by mupan-git** [vuejs/vue-cli#7393] [General]
**1. 我通过以下配置修改了图片输出目录**
**2. 打包后 css 中的 background-image 路径指向了dist下的一张异常的图片地址**

![image](https://github.com/vuejs/vue-cli/assets/37840088/0f47a3ed-9307-4f4b-8f97-597ff28357af)
![image](https://github.com/vuejs/vue-cli/assets/37840088/147317d4-a0d0-44c5-9d2f-311515bad655)
![image](https://github.com/vuejs/vue-cli/assets/37840088/6834b8f4-49be-490e-8ec9-63f34e332c45)

**Comment by mdxiaohu** [comment]
Has it not been resolved yet .
